.github/workflows/pr-build.yaml

```diff
@@ -33,19 +30,10 @@ jobs:
             docker build -t galasabld:${{env.IMAGE_TAG}} --build-arg platform=linux-amd64 -f dockerfiles/galasabld/dockerfile.galasabld .
             docker run --rm galasabld:${{env.IMAGE_TAG}}
     
-        - name: Push galasabld executables
-          uses: actions/upload-artifact@v4
-          with:
-            name: galasabld-${{matrix.config.name}}-${{matrix.config.arch}}-${{ github.event.number }}
-            path: bin/galasabld-${{matrix.config.name}}-${{matrix.config.arch}}
-
     build-upload-openapi2beans:
       name: Build openapi2beans
       runs-on: ubuntu-latest
-      strategy:
-        matrix:
-          config: [{'name':'darwin','arch':'arm64'},{'name':'darwin','arch':'x86_64'},{'name':'linux','arch':'x86_64'}]
-      
+

```

Comment:
The upload step should be part of the PR build too as if you've made changes to one of the executables and you've raised a PR, you can then download the executable with the changes and test it out

Reply:
done